apis/v1alpha2/util/translator/httproute.go

```diff
@@ -14,20 +14,20 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package utils
+package translator
```

Comment:
minor: I see "...er" package names less often than action or group-centered names.  E.g. `translate.PathMatchTypePtr(s)`

Reply:
sounds good, changed in a1e5a468af2cc6e05fef42f425d34c5f4559b92d of #1569